codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -492,13 +476,12 @@ public void onGoAwayRead(ChannelHandlerContext ctx, int lastStreamId, long error
         public void onWindowUpdateRead(ChannelHandlerContext ctx, int streamId, int windowSizeIncrement)
                 throws Http2Exception {
             Http2Stream stream = connection.requireStream(streamId);
-            verifyGoAwayNotReceived();
-            if (stream.state() == CLOSED || shouldIgnoreFrame(stream, false)) {
-                // Ignore frames for any stream created after we sent a go-away.
+
+            if (stream.state() == CLOSED || streamCreatedAfterGoAwaySent(stream)) {
```

Comment:
@buchgr - My choice of words was wrong.  `Now that we allow closed streams in the flow control window...` should have said `Now that we allow closed streams to remain in the priority tree...`...does this make more sense?

> We only care about the connection window getting updated.

+1.  Agreed we don't need to worry about this.  Code is fine as is.